llvm/test/Transforms/Attributor/multiple-offsets-pointer-info.ll

```diff
@@ -15,6 +15,9 @@ define i8 @select_offsets_simplifiable_1(i1 %cnd1, i1 %cnd2) {
 ; CHECK-NEXT:    store i8 23, ptr [[GEP23]], align 4
 ; CHECK-NEXT:    [[GEP29:%.*]] = getelementptr inbounds [1024 x i8], ptr [[BYTES]], i64 0, i64 29
 ; CHECK-NEXT:    store i8 29, ptr [[GEP29]], align 4
+; CHECK-NEXT:    [[GEP9:%.*]] = getelementptr inbounds [1024 x i8], ptr [[BYTES]], i64 0, i64 9
```

Comment:
It seems like it is not removed yet. Expected?

Reply:
Yes, it overlaps with index 7 (due to the i16). More coverage for the case the index doesn't match but there is an overlap.